app/scenes/AppletDetails/AppletDetailsComponent.js

```diff
@@ -72,7 +76,7 @@ class AppletDetailsComponent extends React.Component {
       appletData,
     } = this.props;
 
-    const responseDates = this.getResponseDates() || [];
+    const responseDates = this.getResponseDates();
```

Comment:
This method `getResponseDates` might return `undefined` depending on the age of several documents on the backend. The `|| []` here sets `const responseDates` to an empty Array if that method returns `undefined`. Please do any one of the following before we merge:
1. restore the `|| []` default value handler
2. add better antifragility handling
3. document why this change is necessary